apis/falcon/v1alpha1/falconconfig_types.go

```diff
@@ -16,6 +16,9 @@ type FalconAPI struct {
 	ClientId string `json:"client_id"`
 	// Falcon OAuth2 API Client Secret
 	ClientSecret string `json:"client_secret"`
+	// Falcon Customer ID (CID)
+	// +kubebuilder:validation:Pattern="[0-9a-zA-Z]+-[0-9a-zA-Z]{2}"
```

Comment:
Can you make the regex `^[0-9a-fA-F]{32}-[0-9a-fA-F]{2}$` or `[0-9a-fA-F]{32}-[0-9a-fA-F]{2}`

Reply:
Yes, I can.